test/fixtures/xCartesianTime.js

```diff
@@ -0,0 +1,55 @@
+module.exports = {
+  config: {
+    type: 'line',
+    data: {
+      datasets: [{
+        data: [
+          {x: 1648249200000, y: 24},
```

Comment:
You should use strings here without timezone, so it will be parsed and displayed at the same timezone

Reply:
I'm testing